infinitory/templates/layout.html

```diff
@@ -13,6 +13,7 @@
       <ul>
         <li><a href="{{ path }}index.html" class="backward">Home</a></li>
         <li class="infinitory-logo">∞</li>
+        <li><a href="{{ path }}errors/index.html">Errors</a></li>
```

Comment:
Could you move this to the last position?

Reply:
haha I put it here because "n, r, s" is in alphabetical order